docs/man/man1/oc-cluster-join.1

```diff
@@ -0,0 +1,190 @@
+.TH "OC CLUSTER" "1" " Openshift CLI User Manuals" "Openshift" "June 2016"  ""
+
+
+.SH NAME
+.PP
+oc cluster join \- Join an existing OpenShift cluster
+
+
+.SH SYNOPSIS
+.PP
+\fBoc cluster join\fP [OPTIONS]
+
+
+.SH DESCRIPTION
+.PP
+Add a new node to an existing OpenShift cluster
+
+.PP
+Uses an existing connection to a Docker daemon to start an OpenShift node. You must provide a secret to connect to the master. Before running command, ensure that you can execute docker commands successfully (ie. 'docker ps').
```

Comment:
The secret is a kubeconfig file, that has the ability to request CSR.  Expectation here is that your file contains discovery info (we're not focused on self-hosting in OpenShift until it's had time to bake in Kube, so the discovery bits aren't really necessary).